src/sentry/models/organizationmember.py

```diff
@@ -62,9 +63,19 @@ def get_contactable_members_for_org(self, organization_id: int) -> QuerySet:
 
     def delete_expired(self, threshold: int) -> None:
         """Delete un-accepted member invitations that expired `threshold` days ago."""
+
+        # if SCIM is enabled, let SCIM control creation, modification, and deletion of users
+        orgs_without_scim = [
+            org_id
+            for org_id in AuthProvider.objects.exclude(flags__gt=1).values_list(
+                "organization_id", flat=True
+            )
+        ]
```

Comment:
How many org ids are likely to be returned here?

I think you're likely better off not coercing this to a list. If you just leave this as a queryset then Django will make query as a subquery, rather than needing to bring all the ints back from postgres then send them out again.